scripts/Package_BGA/bga_xilinix.yml

```diff
@@ -0,0 +1,148 @@
+Xilinx_CSG324:
+  description: "Artix-7 BGA, 18x18 grid, 15x15mm package, 0.8mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf, NSMD pad definition Appendix A"
+  additional_tags: "CS324 CSG234"
+  pkg_width: 15
+  pkg_height: 15
+  pitch: 0.8
+  pad_diameter: 0.4
+  mask_margin: 0.05
+  layout_x: 18
+  layout_y: 18
+Xilinx_CSG325:
+  description: "Artix-7 BGA, 18x18 grid, 15x15mm package, 0.8mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf, NSMD pad definition Appendix A"
+  additional_tags: "CS325 CSG235"
+  pkg_width: 15
+  pkg_height: 15
+  pitch: 0.8
+  pad_diameter: 0.4
+  mask_margin: 0.05
+  layout_x: 18
+  layout_y: 18
+Xilinx_SBG484:
+  description: "Artix-7 BGA, 22x22 grid, 19x19mm package, 0.8mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf, NSMD pad definition Appendix A"
+  additional_tags: "SB484 SBG484 SBV484"
+  pkg_width: 19
+  pkg_height: 19
+  pitch: 0.8
+  pad_diameter: 0.4
+  mask_margin: 0.05
+  layout_x: 22
+  layout_y: 22
+Xilinx_RS484:
+  description: "Artix-7 BGA, 22x22 grid, 19x19mm package, 0.8mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf, NSMD pad definition Appendix A"
+  pkg_width: 19
+  pkg_height: 19
+  pitch: 0.8
+  pad_diameter: 0.4
+  mask_margin: 0.05
+  layout_x: 22
+  layout_y: 22
+Xilinx_FTG256:
+  description: "Artix-7 BGA, 16x16 grid, 17x17mm package, 1mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf, NSMD pad definition Appendix A"
+  additional_tags: "FT256 FTG256"
+  pkg_width: 17
+  pkg_height: 17
+  pitch: 1
+  pad_diameter: 0.4
+  mask_margin: 0.05
+  layout_x: 16
+  layout_y: 16
+Xilinx_FBG484:
+  description: "Artix-7 BGA, 22x22 grid, 23x23mm package, 1mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf, NSMD pad definition Appendix A"
+  additional_tags: "FB484 FBG484 FBV484"
+  pkg_width: 23
+  pkg_height: 23
+  pitch: 1
+  pad_diameter: 0.53
+  mask_margin: 0.05
+  layout_x: 22
+  layout_y: 22
+Xilinx_FGG484:
+  description: "Artix-7 BGA, 22x22 grid, 23x23mm package, 1mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf, NSMD pad definition Appendix A"
+  additional_tags: "FG484 FGG484"
+  pkg_width: 23
+  pkg_height: 23
+  pitch: 1
+  pad_diameter: 0.5
+  mask_margin: 0.05
+  layout_x: 22
+  layout_y: 22
+Xilinx_RB484:
+  description: "Artix-7 BGA, 22x22 grid, 23x23mm package, 1mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf, NSMD pad definition Appendix A"
+  pkg_width: 23
+  pkg_height: 23
+  pitch: 1
+  pad_diameter: 0.5
```

Comment:
Should be 0.53mm.

Reply:
Wow, good catch.
Will be updated.
I will be more careful in the future!